test/e2e/pager.test.js

```diff
@@ -108,6 +109,26 @@ describe('vl-pager', async () => {
         await assert.eventually.isFalse(pager.isPaginationDisabled());
     });
 
+    it('Als gebruiker zie ik dat de pager elke keer als er data verandert een event uitstuurt', async() => {
+        const pager = await vlPagerPage.getPagerEventListener();
+        const logElement = await vlPagerPage.getPagerEventListenerLog()
+        await assert.eventually.lengthOf(logElement.findElements(By.css("span")), 0);
+        await pager.goToNextPage();
+        await assert.eventually.equal(lastLogLine(logElement), '{"currentPage":"2","totalPage":10,"itemsPerPage":10,"totalItems":100}');
+        await pager.goToPreviousPage();
+        await assert.eventually.equal(lastLogLine(logElement), '{"currentPage":"1","totalPage":10,"itemsPerPage":10,"totalItems":100}');
+        await pager.goToPage(2);
+        await assert.eventually.equal(lastLogLine(logElement), '{"currentPage":"2","totalPage":10,"itemsPerPage":10,"totalItems":100}');
+    });
+    
+    async function lastLogLine(logElement) {
+        const logLines = await logElement.findElements(By.css("span"));
+        if (logLines.length > 0) {
+            const lastLogLine = logLines[logLines.length - 1];
+            return lastLogLine.getText();
+        }
+    }
```

Comment:
Indien mijn eerste change request goedgekeurd wordt:
```suggestion
    it('Als gebruiker zie ik dat de pager elke keer als er data verandert een event uitstuurt', async() => {
        const pager = await vlPagerPage.getPagerEventListener();
        const logElement = await vlPagerPage.getPagerEventListenerLog()
        await assert.eventually.lengthOf(logElement.findElements(By.css("span")), 0);
        await pager.goToNextPage();
        await assert.eventually.equal(logElement, '{"currentPage":"2","totalPage":10,"itemsPerPage":10,"totalItems":100}');
        await pager.goToPreviousPage();
        await assert.eventually.equal(logElement, '{"currentPage":"1","totalPage":10,"itemsPerPage":10,"totalItems":100}');
        await pager.goToPage(2);
        await assert.eventually.equal(logElement, '{"currentPage":"2","totalPage":10,"itemsPerPage":10,"totalItems":100}');
    });
```

Reply:
logElement.getText() overal wel